derived-env

```diff
@@ -4,16 +4,18 @@
 # automatically sourced into setup-env
 
 # Octarine AWS JupyterHub setup
-export ACCOUNT=${ACCOUNT_ID}.dkr.ecr.us-east-1.amazonaws.com
-export ECR_ACCOUNT=${CENTRAL_ECR_ACCOUNT_ID}.dkr.ecr.us-east-1.amazonaws.com
+#export ACCOUNT=${ACCOUNT_ID}.dkr.ecr.us-east-1.amazonaws.com
+#export ECR_ACCOUNT=${CENTRAL_ECR_ACCOUNT_ID}.dkr.ecr.us-east-1.amazonaws.com
 
 if [ "$USE_CENTRAL_ECR" = true ]
 then
-    export ECR_ACCOUNT_TO_USE=$ECR_ACCOUNT
+    export ECR_REGISTRY=${CENTRAL_ECR_ACCOUNT_ID}.dkr.ecr.us-east-1.amazonaws.com
```

Comment:
Is ECR_REGISTRY used anywhere or is it a loose end?

Reply:
I will double check...

Reply:
Yes, it's used in a couple of `image-*` scripts.